plugin/trino-hudi/src/test/resources/hudi-testing-data/hudi_multi_pt_mor.md

```diff
@@ -0,0 +1,60 @@
+## Create script
+Revision: 6f65998117a2d1228fc96d36053bd0d394499afe
+
+```scala
+test("Create MOR table with multiple partition fields with multiple types") {
+  withTempDir { tmp =>
+    val tableName = "hudi_multi_pt_mor"
+    // Save current session timezone and set to UTC for consistency in test
+    val originalTimeZone = spark.conf.get("spark.sql.session.timeZone")
+    spark.conf.set("spark.sql.session.timeZone", "UTC")
+
+    spark.sql(
+      s"""
+         |CREATE TABLE $tableName (
+         |  id INT,
+         |  name STRING,
+         |  price DOUBLE,
+         |  ts LONG,
+         |  -- Partition Fields --
+         |  category STRING,            -- Partition field 1: STRING
+         |  year INT,                   -- Partition field 2: INT
+         |  event_date DATE,            -- Partition field 3: DATE
+         |  batch_id BIGINT,            -- Partition field 4: BIGINT
+         |  metric_value DECIMAL(10,2), -- Partition field 5: DECIMAL
+         |  event_timestamp TIMESTAMP,  -- Partition field 6: TIMESTAMP
```

Comment:
Directly using DECIMAL or TIMESTAMP field for partitioning without transformation might be rare.

Reply:
Agree, for the sake of test coverage, i added it, but am not supporting it directly as there are some write issues here.

Reply:
Sg